web/src/reducers/activities.js

```diff
@@ -1,27 +1,23 @@
-import { ADD_ACTIVITY, UPDATE_ACTIVITY } from '../actions/activities';
+import u from 'updeep';
+
+import {
+  ADD_ACTIVITY,
+  ADD_ACTIVITY_GOAL,
+  UPDATE_ACTIVITY
+} from '../actions/activities';
+
+const newGoal = () => ({ desc: '', obj: '' });
```

Comment:
In this code, there's one "objective" per goal.  In the current data model, there are multiple objectives per goal.  Was this a change from product?  If so, we should also make an issue to update the data model too.  If not, we can hash out the redux specifics when we go to hook everything up.

Reply:
this was a change driven by a conversation with nick a couple weeks ago